packages/table-dev-app/src/mutableTable.tsx

```diff
@@ -106,7 +106,7 @@ const LONG_TEXT_WORD_SPLIT_REGEXP = /.{1,5}/g;
 const LARGE_JSON_PROP_COUNT = 3;
 const LARGE_JSON_OBJECT_DEPTH = 2;
 
-const CELL_CONTENT_GENERATORS = {
+const CELL_CONTENT_GENERATORS: { [name: string]: (ri: number, ci: number) => string | object } = {
```

Comment:
Make an interface for this maybe? Or some piece of it?

Reply:
meh it's just dev code, used exactly once